packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -349,8 +349,8 @@ let gridTrackSize = (value: GridTrackSize) => {
 const transitionProperty = {
   // var(--gp) is generated by the backgroundImage property when setting a gradient.
   // It includes a list of all of the custom properties used for each color stop.
-  default: 'color, background-color, var(--gp), border-color, text-decoration-color, fill, stroke, opacity, box-shadow, transform, translate, scale, rotate, filter, backdrop-filter',
-  colors: 'color, background-color, var(--gp), border-color, text-decoration-color, fill, stroke',
+  default: 'color, background-color, var(--gp, color), border-color, text-decoration-color, fill, stroke, opacity, box-shadow, transform, translate, scale, rotate, filter, backdrop-filter',
```

Comment:
is --gp still correct?

Reply:
yeah, the missing fallback was a bug before. when `--gp` is not defined, this resulted in `all` properties transitioning instead of only the listed ones. Now I made it fall back to something already in the list.